home/ivp-planning/advanced-tools/path-tool.md

```diff
@@ -1,102 +1,80 @@
+### Editing Path Distance and Angles
 
-![](../../../.gitbook/assets/iVP\_path\_tool\_draw\_path\_path\_panel.jpg)
+Existing paths can be further edited to adjust the individual lengths between corner points. To do so, highlight one of your corner points and edit either the "Distance to previous point" or "Distance to next point" options. It is possible to edit both options at the same time.
 
-![](../../../.gitbook/assets/iVP\_path\_tool\_list\_right\_click\_menu\_measurements.jpg)
+![](../../../.gitbook/assets/iVP_path_tool_edit_corner_distance.jpg)
 
-## Area measurements:
+Additionally, you can edit the individual axis angles of a path. The blue line serves as the global Y-axis, providing a better reference point when adjusting your angles.  
+The angles are measured based on the line between two points, where one of these points is the current point and the other is either the previous or the next point. The angle is formed between this line and the vertical axis (blue line)
 
-Besides lengths and angles, areas are measurable as well. To activate this option, **check the 'Area' box** under the measurements foldout of the [properties](../user-interface/the-info-panel.md), followed by a click in **"Area"**.
+![](../../../.gitbook/assets/iVP_path_tool_edit_axis_distance.jpg)
+## Measurement display options
 
-![](../../../.gitbook/assets/iVP\_path\_tool\_draw\_path\_path\_panel.jpg)
+You can toggle the display of **edge lengths**, **angle** and **area** information for each path by right-clicking on them and selecting the corresponding option in the context menu.
 
-![](../../../.gitbook/assets/iVP\_path\_tool\_list\_right\_click\_menu\_area.jpg)
+![](../../../.gitbook/assets/iVP_path_tool_context_menu.jpg)
 
-The path will now be filled with a stripe pattern and an area size will be displayed in the [2D-panel](../user-interface/the-2d-panel.md) above the path name.
+Alternatively it is possible to hide or show all lengths or angles by changing the entry in the measurements of the [properties panel](../user-interface/the-info-panel.md) and subsequently clicking on **edge lengths**, **angles** or **area**.
 
-{% hint style="info" %}
-The "Area" option also affects the path in 3D-panel so that it can be used to highlight specific areas.
-{% endhint %}
-
-![](../../../.gitbook/assets/iVP\_path\_tool\_draw\_path\_area.jpg)
-
-## Path renaming:
-
-Paths can be [renamed](../machines/renaming-objects-and-folders.md) at any time just like any other object.
-
-{% hint style="info" %}
-There are no limitations concerning the name length of a path or concerning special characters like the umlauts "ä", "ü" and "ö".
-{% endhint %}
+![](../../../.gitbook/assets/iVP_path_tool_properties_window_2d.jpg)
 
-![](../../../.gitbook/assets/iVP\_path\_tool\_draw\_path\_path\_panel.jpg)
+## Ground Markings
```

Comment:
Given that fence and room have it's own pages, this one should be on a separate page too.